api/envoy/extensions/matching/common_inputs/network/v3/network_inputs.proto

```diff
@@ -95,3 +96,23 @@ message TransportProtocolInput {
 // [#extension: envoy.matching.inputs.application_protocol]
 message ApplicationProtocolInput {
 }
+
+// Input that matches the authentication attributes for a downstream.
+// [#extension: envoy.matching.inputs.authenticated]
+message AuthenticatedInput {
+  enum Field {
+    // The URIs in the SAN field of the certificate. Certificate with multiple URIs will be
+    // concatenated into a ','-delimited list.
+    UriSan = 0;
```

Comment:
Enum values should be all-caps with underscores, as per:

https://cloud.google.com/apis/design/naming_convention#enum_names

Also, see the guidance in https://github.com/envoyproxy/envoy/blob/main/api/STYLE.md about the 0 value in enums.

Reply:
Thanks, fixed.